src/main/antlr4/imports/mysql_idents.g4

```diff
@@ -19,8 +19,8 @@ name: ( id | tokens_available_for_names | INTEGER_LITERAL);
 id: ( IDENT | QUOTED_IDENT );
 literal: (float_literal | integer_literal | string_literal | NULL | TRUE | FALSE);
 
-float_literal: INTEGER_LITERAL? '.' INTEGER_LITERAL;
-integer_literal: INTEGER_LITERAL;
+float_literal: '-'? INTEGER_LITERAL? '.' INTEGER_LITERAL;
```

Comment:
Looks like this also allows stuff like `-1.-123` - is that a problem or won't it matter because we'll never actually get that from mysql?